src/components/CustomInput.jsx

```diff
@@ -1,30 +1,39 @@
 import React, { useState } from "react";
-import { useDispatch } from "react-redux";
-import { addTask } from "../redux/actions/inputAction";
+import { Button, Input } from "semantic-ui-react";
+import { createTaskRequest } from "../api";
+import "semantic-ui-css/semantic.min.css";
 
 export default function CustomInput(props) {
```

Comment:
Name of component should be self-explanatory

Reply:
fixed